File: DSA_Problems/sample.py
# Function to test
def add(a, b):
    return a + b

# Test case 1
if add(1, 2) == 3:
    print("Test case 1 passed")
else:
    print("Test case 1 failed")

# Test case 2
if add(0, 0) == 0:
    print("Test case 2 passed")
else:
    print("Test case 2 failed")

# Test case 3
if add(-1, 1) == 0:
    print("Test case 3 passed")
else:
    print("Test case 3 failed")

File: DSA_Problems/sample.cpp
#include <iostream>
using namespace std;

// Function to test
int add(int a, int b) {
    return a + b;
}

int main() {
    // Test case 1
    if(add(1, 2) == 3) {
        cout << "Test case 1 passed\n";
    } else {
        cout << "Test case 1 failed\n";
    }

    // Test case 2
    if(add(0, 0) == 0) {
        cout << "Test case 2 passed\n";
    } else {
        cout << "Test case 2 failed\n";
    }

    // Test case 3
    if(add(-1, 1) == 0) {
        cout << "Test case 3 passed\n";
    } else {
        cout << "Test case 3 failed\n";
    }

    return 0;
}